lib/pengine/unpack.c

```diff
@@ -89,14 +89,20 @@ pe_fence_node(pe_working_set_t * data_set, node_t * node, const char *reason)
         set_bit(node->details->remote_rsc->flags, pe_rsc_failed);
 
     } else if (is_baremetal_remote_node(node)) {
-        if(pe_can_fence(data_set, node)) {
-            crm_warn("Node %s will be fenced %s", node->details->uname, reason);
+        resource_t *rsc = node->details->remote_rsc;
+
+        if ((rsc != NULL) && (is_set(rsc->flags, pe_rsc_managed))) {
+            if(pe_can_fence(data_set, node)) {
```

Comment:
space after if looks better - true.
but as this is old code and the same pattern repeats throughout the code (at least this file)
I guess it would be cleaner to do that in an extra commit after we are through with that
covering all occurrences of the same pattern at least.
Before would be fine of cause as well but is more effort.